pyccel/ast/variable.py

```diff
@@ -401,6 +410,12 @@ def is_ndarray(self):
         return isinstance(self.dtype, (NativeInteger, NativeBool,
                           NativeFloat, NativeComplex))
 
+    @property
+    def is_tuple(self):
+        """user friendly method to check if the variable is a tuple:
+        """
+        return self._is_tuple
```

Comment:
Shouldn't this return `False`? It should only be `True` if it is overwritten by `TupleVariable.is_tuple`